css/min-width-768px-style.css

```diff
@@ -0,0 +1,129 @@
+.headline {
+  background-image: none;
+  align-content: center;
+  padding: 1px 20% 250px 20%;
+}
+
+.headline > * {
+  margin: 12px !important;
+}
+
+.toolbar {
+  justify-content: center;
+}
+
+#port-toolbar:hover {
+  filter: contrast(200%);
+}
+
+#abou-toolbar:hover {
+  filter: contrast(200%);
+}
+
+#cont-toolbar:hover {
+  filter: contrast(200%);
+}
+
+.hole-page {
+  background-size: 100% 100%;
+  background-position: center top;
+  background-repeat: no-repeat;
+  background-image: url(../images/background/min-992px-main-page-background-image.svg);
+}
+
+.welcome {
+  display: none;
+}
+
+#drop-image {
+  display: none;
+}
+
+.desktop-toolbar {
+  display: flex;
+  padding: 16px;
+  justify-content: center;
+}
+
+.desktop-toolbar > * {
+  padding: 10px 12px;
+  list-style-type: none;
+}
+
+.desktop-toolbar > * > a {
+  text-decoration: none;
+  color: #344563;
+}
+
+.work-title {
+  margin-bottom: 30px;
+}
+
+.work-title img {
+  display: none;
+}
+
+.about-description {
+  background-image: url(../images/background/about-me-background-image-2.svg);
+  background-position: center top;
+  background-repeat: no-repeat;
+}
+
+.about-me-text {
+  max-width: 55%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.abilities {
+  grid-template-columns: 370px 370px;
+}
+
+.contact {
+  background-image: url(../images/background/desktop-contact-me-background-image.png);
+  margin-top: 10vw;
+}
```

Comment:
- [ ] Thank you for implementing the previous reviewers' comments for the desktop version. However, I notice also your mobile version could be improved on its alignment.  the "I'm always interested..." text should be align vertically with the edge of the `form`. You can kindly implement this change. This will ensure that you are matching the design in both desktop and mobile versions.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/144515715-f202e440-f3aa-4ebc-be09-e384e0f15276.png)

## Figma Design (notice the alignment of the text and inputs)
 
![image](https://user-images.githubusercontent.com/18730106/144515799-a987d121-7b65-41a9-b5b2-f80224815f87.png)